test/svid/jwtsvid/test_jwt_svid.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import datetime
+from calendar import timegm
+import jwt
+
+from pyspiffe.svid.jwt_svid import (
+    JwtSvid,
+    INVALID_INPUT_ERROR,
+)
+from pyspiffe.svid.exceptions import (
+    TokenExpiredError,
+    JwtSvidError,
+    InvalidClaimError,
+)
+
+"""
+    parse_insecure tests
+"""
+
+
+@pytest.mark.parametrize(
+    'test_input_token,test_input_audience, expected',
+    [
+        ('', [], INVALID_INPUT_ERROR.format('token cannot be empty')),
+        ('', None, INVALID_INPUT_ERROR.format('token cannot be empty')),
+        (None, [], INVALID_INPUT_ERROR.format('token cannot be empty')),
+        (None, None, INVALID_INPUT_ERROR.format('token cannot be empty')),
+    ],
+)
+def test_invalid_input_parse_insecure(test_input_token, test_input_audience, expected):
+    with pytest.raises(ValueError) as exception:
+        JwtSvid.parse_insecure(test_input_token, test_input_audience)
+
+    assert str(exception.value) == expected
+
+
+@pytest.mark.parametrize(
+    'test_input_token,test_input_audience, expected',
+    [
+        (
+            jwt.encode(
+                {
+                    'sub': 'spiffeid://somewhere.over.the',
+                    'exp': timegm(
+                        (
+                            datetime.datetime.utcnow() + datetime.timedelta(hours=72)
+                        ).utctimetuple()
+                    ),
+                },
+                'secret',
+                headers={'alg': 'RS256', 'typ': 'JOSE'},
+            ),
+            ["spire"],
+            InvalidClaimError._text.format('aud'),
```

Comment:
nit: Doesn't work  `InvalidClaimError('aud').text` here?
Accessing the private property directly looks at least weird to me.

Reply:
That is a great catch, Andres! Thanks for raising it. I've fixed it.